images/hats/wing.svg

```diff
@@ -1 +1,10 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 371.81 234.76" width="1em" height="1em"><defs><style>.cls-1{fill:#666666;}</style></defs><g id="Layer_2" data-name="Layer 2"><g id="Layer_1-2" data-name="Layer 1"><path class="cls-1" d="M371.81,234.42h0s-153.27-57.29-185.9-54.08h0C154.36,177.24,0,234.76,0,234.76H0L158.55,101.69c.6-.49-.39,1.87-.23,1.12L186.24,0h0l26.84,101.37"/></g></g></svg>
\ No newline at end of file
+<svg width="12" height="9" viewBox="0 0 12 9" fill="none" xmlns="http://www.w3.org/2000/svg">
+<g clip-path="url(#clip0_25_59)">
+<path d="M6 0L12 9L6 7.5L0 9L6 0Z" fill="#FF0000"/>
+</g>
+<defs>
+<clipPath id="clip0_25_59">
+<rect width="12" height="9" fill="white"/>
```

Comment:
This doesn't look right 🤔

Reply:
That is the frame. It appears to not be rendering. I'm not quite sure why.

Reply:
Even on dark VSCode?

Reply:
Oh right it's used as a clip path. So presumably just clips the shape if it extends outside the view box